src/wagtail_factories/factories.py

```diff
@@ -146,3 +147,31 @@ class Meta:
 
     title = "A document"
     file = factory.django.FileField()
+
+
+class RedirectFactory(DjangoModelFactory):
```

Comment:
@JakubMastalerz I tweaked the `RedirectFactory` you created in `wagtail-grapple` to add more comments.

The example here: https://github.com/wagtail/wagtail-factories/issues/82 is a better one to use.

Reply:
I've added the reference to `Redirect.clean()`. I believe the `RedirectFactory` from `wagtail-grapple` is more complex than it has to be. Since we're calling `clean()` not `full_clean()` we shouldn't have to handle those validation errors.